include/SDL3/SDL_stdinc.h

```diff
@@ -49,11 +49,15 @@
 #include <SDL3/SDL_platform_defines.h>
 
 #include <stdarg.h>
+#if defined (HAVE_STDINT_H)
```

Comment:
Does ngage sdk really not have stdint.h and wchar.h? If so, why is it not updated for that?

Reply:
I updated only the C compiler and wrote some missing libc functions to be able to compile SDL3. I had to touch as little as possible and I'm glad that we found a combo that finally produced working results.

So yes, we do not have stdint.h and wchar.h

Reply:
I think you should consider adding a minimum c99-compatible `stdint.h` to the ngage toolchain.
It will make your life a lot easier.
`stdint.h` of glibc is only around 300 lines (including a lot of comments).

Reply:
Done!